update.sh

```diff
@@ -17,6 +17,8 @@ packages="$packages e2fsprogs"
 packages="$packages f2fs-tools"
 packages="$packages gpgv"
 packages="$packages raspbian-archive-keyring"
+packages="$packages wpasupplicant"
+packages="$packages firmware-realtek"
```

Comment:
Could you order them alphabetically? 
It's probably also a good idea to also include `firmware-linux-free`, `firmware-linux-nonfree` and probably various others.